docs/api/en/math/Line3.html

```diff
@@ -113,7 +113,8 @@ <h3>[method:Line3 set]( [param:Vector3 start], [param:Vector3 end] )</h3>
 		[page:Vector3 start] - set the [page:.start start point] of the line.<br />
 		[page:Vector3 end] - set the [page:.end end point] of the line.<br /><br />
 
-		Sets the start and end values by copying the provided vectors.
+		Sets the start and end values by copying the provided vectors.<br>
+		Please note that this method only copies the values from the given objects.
```

Comment:
This line seems redundant - the first sentence already says that the vectors are copied.